src/components/BooksList.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { connect, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import actions from '../actions/index';
+import Book from '../containers/Book';
+import CategoryFilter from '../containers/CategoryFilter';
+
+const BooksList = ({ changeFilter }) => {
+  const books = useSelector(state => state.books);
+  const filterState = useSelector(state => state.filter);
+  const handleFilterChange = e => {
+    changeFilter(e.target.value);
+  };
+
+  const filteredBooks = () => {
```

Comment:
Filtering should be a redux action, no? 🤔 
Why is the logic here?

Reply:
It's part of the project requirement to change the `filter` state with redux. Using redux to also filter would make changing the `filter` state redundant